mithril-infra/assets/tools/pool/create-keys.sh

```diff
@@ -62,9 +62,20 @@ CARDANO_CLI_CMD node issue-op-cert \
 --out-file ${POOL_ARTIFACTS_DIR}/opcert.cert
 
 ### Create a registration certificate
-CARDANO_CLI_CMD stake-address registration-certificate \
---stake-verification-key-file ${POOL_ARTIFACTS_DIR}/stake.vkey \
---out-file ${POOL_ARTIFACTS_DIR}/stake.cert
+if [ "${CARDANO_ERA}" == "conway"]; then
```

Comment:
Is it only for conway or this version and future ones ? 
There is a risk that a new era will be treated as a legacy one (else statement)

Reply:
You're right, but this code will be simplified once the Conway era has been activated, and the `else` part will be removed :+1: